src/components/Dialogs/QueueModal.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useLayoutEffect, useMemo, useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import cx from 'classnames';
+import { filter, map, reduce, throttle } from 'lodash';
+import { Icon } from '@mdi/react';
+import { mdiAlertCircleOutline, mdiCheckboxMarkedCircleOutline, mdiChevronRight, mdiCircleOutline, mdiHelpCircleOutline, mdiLoading, mdiRun } from '@mdi/js';
+import { useEventCallback } from 'usehooks-ts';
+
+import ModalPanel from '@/components/Panels/ModalPanel';
+import { useGetQueueOperationMutation, useLazyGetQueueItemsQuery } from '@/core/rtkQuery/splitV3Api/queueApi';
+import { RootState } from '@/core/store';
+import { QueueItemType } from '@/core/types/api/queue';
+import Input from '../Input/Input';
+import MenuButton from '../Utilities/Unrecognized/MenuButton';
+
+type QueueName = keyof typeof names;
```

Comment:
Usage of variable before it's declared.
How did eslint not complain? We'll never know
fix it